internal/actions/sync_stack.go

```diff
@@ -69,17 +56,35 @@ type StackSyncState struct {
 	Config StackSyncConfig `json:"config"`
 }
 
-// Performs stack sync on all branches in branchesToSync.
+type (
+	SyncStackOpt  func(*syncStackOpts)
+	syncStackOpts struct {
+		skipNextCommit bool
+	}
+)
+
+func WithSkipNextCommit() SyncStackOpt {
```

Comment:
I'm not 100% on this but I thought doing the functional option style was a better choice than putting it in the state (since the state gets persisted to disk, but `--skip` is fundamentally a one-time thing.

Reply:
Using this style seems to me too much for `internal` package, but if you want to use this sure.